tools/postgres/apply_evolutions.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash          
+set -Eeuo pipefail
+
+# find schema version
+schema_version=`psql -A  -U postgres -h localhost --dbname="webknossos" -t -c "SELECT * FROM webknossos.releaseinformation"`
+echo "Schema version: ${schema_version}"
+
+# assert that the found schema version is a number
+re='^[0-9]+$'
+if ! [[ $schema_version =~ $re ]] ; then
+   echo "Error: Schema versoin is not a number" >&2; exit 1
+fi
+
+# get list of schemas to apply 
+files=""
+for entry in "../../conf/evolutions"/*
```

Comment:
How can I get the absolute path to this folder?
When I run `./apply_evolutions.sh` from the right directory everything works, but when I run `yarn apply-evolutions` it seems like it does not find the evolutions.

Reply:
you could make the path be relative to the script’s directory
https://stackoverflow.com/a/246128/7564727

Reply:
We usually use `scriptdir="$(dirname "$0")"` to get the path of the script, and the use everything relatively to that. An example is https://github.com/scalableminds/webknossos/blob/master/tools/postgres/ensure_db.sh